### DISCUSSION TITLE
如何将Avalonia UI生成的模板升级到.NET8？

### DISCUSSION THREAD

**Discussion by AFunDog** [BYJRK/DotNet-Discussions#295] [其他框架]
![image](https://github.com/BYJRK/DotNet-Discussions/assets/50763230/64b6ffb5-3134-445b-afc0-7b0e981ad443)
如图，我用VS2022的Avalonia UI模板生成项目，但初始项目是.NET7版本的。我想将其版本改为.NET8，但直接修改会出现各种错误，有没有大佬能教教我如何升级到.NET8版本。🎈

**Comment by BYJRK** [comment]
你的 Avalonia 的模板应该是用 dotnet cli 安装的吧？在控制台运行：

```bash
dotnet new update
```

试试呢

**Reply by AFunDog** [reply]
我尝试了一下发现如下结果
![image](https://github.com/BYJRK/DotNet-Discussions/assets/50763230/9fbf2bd6-6839-4be5-a4e2-2581b3599394)
我实际上模板是VS2022里的扩展安装的，难道那不是最新的吗？或者有其他的最新的安装途径

**Reply by AFunDog** [reply]
![image](https://github.com/BYJRK/DotNet-Discussions/assets/50763230/493bf662-a09b-4838-a197-666fbbe26bb4)
非常感谢大佬的帮助，我在Avalonia的官网找到了安装模板的命令，可以实现创建.NET8的项目。从扩展商店安装的模板并不正确，我已经删除了。
